dist/releases/4.19/config.toml

```diff
@@ -536,3 +536,10 @@ files = [
   "/usr/sbin/build-locale-archive",
   "/usr/sbin/ldconfig"
 ]
+
+[[payload.ocp-virt-validation-checkup-rhel9-container.ignore]]
+error = "ErrGoNotCgoEnabled"
+files = [
+  "/test/kv.test",
```

Comment:
Given the path name, are these only used for testing purposes? If so, that's good information to also document in the commit message for archeological purposes.

Reply:
Yes, they are used only for testing purposes. But still, I would like to remark that this is caused by a false positive, the binaries are indeed CGO_ENABLED.